### PR TITLE
[Jormungandr] Direct path with p+r

### PR DESCRIPTION
**Pull request by xlqian**
- [x]  add some tests...

https://jira.kisio.org/browse/TRR-93

In order to integrate external calculators...

### CODE REVIEW

File: source/jormungandr/jormungandr/instance.py [hunk 1]
```diff
@@ -719,6 +718,16 @@ def get_street_network(self, mode, request):
             streetnetwork_backend_conf = getattr(self, column_in_db)
             return self._streetnetwork_backend_manager.get_street_network_db(self, streetnetwork_backend_conf)
 
+    def get_street_network(self, mode, request):
+        if mode != fallback_modes.FallbackModes.car.name:
+            return self._get_street_network(mode, request)
+
+        walking_service = self._get_street_network(fallback_modes.FallbackModes.car.name, request)
```
Comment (benoit-bst) [inline source/jormungandr/jormungandr/instance.py]: why this is not `fallback_modes.FallbackModes.walking.name` ?

File: source/jormungandr/jormungandr/street_network/utils.py [hunk 2]
```diff
@@ -59,3 +59,24 @@ def make_speed_switcher(req):
         FallbackModes.ridesharing.name: req['car_no_park_speed'],
         FallbackModes.taxi.name: req['taxi_speed'],
     }
+
+
+PARK_RIDE_VALUES = {'yes'}
```
Comment (benoit-bst) [inline source/jormungandr/jormungandr/street_network/utils.py:64]: if you want to declare here, please erase this little cute https://github.com/CanalTP/navitia/pull/3228/files#diff-a16a0bc9cd0342e4e2b429a601790fe0R46

File: source/jormungandr/jormungandr/street_network/utils.py [hunk 3]
```diff
@@ -59,3 +59,24 @@ def make_speed_switcher(req):
...
+
+
+def pick_up_park_ride_car_park(pt_objects):
```
Comment (benoit-bst) [inline source/jormungandr/jormungandr/street_network/utils.py:67]: same here https://github.com/CanalTP/navitia/pull/3228/files#diff-a16a0bc9cd0342e4e2b429a601790fe0R132
the function already exists

File: source/jormungandr/jormungandr/instance.py (hunk 1 above)
Comment (xlqian) [reply source/jormungandr/jormungandr/instance.py]: copy paste is evil !

File: source/jormungandr/jormungandr/street_network/utils.py (hunk 2 above)
Comment (xlqian) [reply source/jormungandr/jormungandr/street_network/utils.py:64]: you're right!

**Review by kadhikari — APPROVED** [review]
Seems fine to me.

**Review by pbench — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**